tests/test_bgp_af.rb

```diff
@@ -305,7 +435,7 @@ def test_additional_paths
 
   ##
   ## additional_paths_selection route-map
-  ##
+  ## 
```

Comment:
git and rubocop should have complained about trailing whitespace here - are you running rubocop? do you have the Git hooks installed?